### PR TITLE
i18n(es): update astro-components

### PR DESCRIPTION
**Pull request by kevinzunigacuellar**
#### What kind of changes does this PR include?

- updated content
- Translated content

#### Description

- update `astro-components.md` #1253

### CODE REVIEW

File: src/pages/es/core-concepts/astro-components.md [hunk 1]
```diff
@@ -6,18 +6,20 @@ description: Una introducción a la sintaxis de los componentes .astro.
...
+La sintaxis del componente de Astro es un superconjunto de HTML. Fue [diseñada para sea familiar para alguien con experiencia en HTML o JSX](/es/comparing-astro-vs-other-tools/#astro-vs-jsx), además es compatible con componentes y expresiones de JavaScript. Puedes identificar los componentes Astro por su extensión: `.astro`.
 
-Lo más importante acerca de los componentes de Astro es que **se renderizan a HTML durante la compilación final**. Aún si posees código Javascript dentro de tus componentes, este código solo se ejecuta al compilar tu projecto, siendo removido de la página final que se envía al usuario. El resultado es un sitio web más rápido y sin rastros de Javascript.
+Los componentes de Astro son extremadamente flexibles. Un componente de Astro puede contener **UI reutilizable**, tal como encabezados o una tarjeta de perfil. También puede contener un fragmento pequeño de HTML, o una colección de etiquetas `<meta>` para facilitar nuestro trabajo con el SEO. Los componentes de Astro también pueden contener un layout de la página.
```
Comment (threkk) [inline src/pages/es/core-concepts/astro-components.md:13]:
```suggestion
Los componentes de Astro son extremadamente flexibles. Un componente de Astro puede contener **UI reutilizable**, tal como encabezados o una tarjeta de perfil. También puede contener un fragmento pequeño de HTML, o una colección de etiquetas `<meta>` para facilitar nuestro trabajo con el SEO. Los componentes de Astro también pueden contener el layout de una página.
```

File: src/pages/es/core-concepts/astro-components.md [hunk 2]
```diff
@@ -43,45 +44,44 @@ import Button from './Button.astro';
...
 const {title} = Astro.props;
 // Consultar datos externos, de una API privada o base de datos
-const data = await fetch('API_URL_SECRETA/users').then(r => r.json());
+const datos = await fetch('API_URL_SECRETA/users').then(r => r.json());
 ---
 <!-- ¡Tu maquetado va aquí! -->
 ```
 
-La valla de código está diseñada para garantizar que el código Javascript que escribes adentro se encuentre "encapsulado". Este código no va a filtrarse a tu aplicación de frontend, o caer en las manos del usuario. Puedes escribir código que sea costoso o sensible (como una llamada a tu base de datos privada) sin preocuparte por que estos datos sensibles lleguen al navegador del usuario.
+La valla de código está diseñada para garantizar que el código JavaScript que escribes adentro se encuentre "encapsulado". Este código no se filtrará a tu aplicación, o caer en las manos del usuario. Puedes escribir código que sea costoso o sensible (como una llamada a la base de datos) sin preocuparte por que estos datos sensibles lleguen al navegador del usuario.
```
Comment (threkk) [inline src/pages/es/core-concepts/astro-components.md:70]:
```suggestion
La valla de código está diseñada para garantizar que el código JavaScript que escribes adentro se encuentre "encapsulado". Este código no se filtrará a tu aplicación, o llegará al usuario final. Puedes escribir código que sea costoso o sensible (como una llamada a la base de datos) sin preocuparte por que estos datos sensibles lleguen al navegador del usuario.
```

File: src/pages/es/core-concepts/astro-components.md [hunk 3]
```diff
@@ -180,33 +192,71 @@ const items = ["Perro", "Gato", "Mono"];
...
+#### Atributos
+
+En Astro, utiliza el formato estándar `kebab-case` para todos los atributos HTML en lugar del `camelCase` que se usa en JSX. Esto incluso funciona para `class`, que no es compatible con React.
+
+```jsx del={1} ins={2} title="example.astro"
+<div className="box" dataValue="3" />
+<div class="box" data-value="3" />
+```
+
+#### Modifying `<head>`
```
Comment (threkk) [inline src/pages/es/core-concepts/astro-components.md:217]:
```suggestion
#### Modificando `<head>`
```

File: src/pages/es/core-concepts/astro-components.md [hunk 4]
```diff
@@ -180,33 +192,71 @@ const items = ["Perro", "Gato", "Mono"];
...
+
+En JSX, existen bibliotecas especiales para ayudarlo a administrar la etiquetas `<head>` de la página. Esto no es necesario en Astro. Escriba `<head>` y su contenido en un layout de nivel superior.
```
Comment (threkk) [inline src/pages/es/core-concepts/astro-components.md:219]: Although `biblioteca` is the translation for `library`, it is more common to use `librería` (a false friend, which means `book shop`). I have no proof for this besides my personal impression.

```suggestion
En JSX, existen librerias especiales para ayudarlo a administrar la etiquetas `<head>` de la página. Esto no es necesario en Astro. Escriba `<head>` y su contenido en un layout de nivel superior.
```

**Review by threkk — COMMENTED** [review]
I added some comments about things that sound weird :)

**Comment by kevinzunigacuellar** [comment]
Go to go! ✅

**Comment by Yan-Thomas** [comment]
Thanks @kevinzunigacuellar, @threkk, good job! 💜

**Pull request merged**